src/policy/fees.h

```diff
@@ -225,19 +225,19 @@ class CBlockPolicyEstimator
     void processBlock(unsigned int nBlockHeight,
                       std::vector<CTxMemPoolEntry>& entries, bool fCurrentEstimate);
 
-    /** Process a transaction confirmed in a block*/
+    /** Process a transaction confirmed in a block */
```

Comment:
NACK on those. Doxygen doesn't care about such white space. IIrc we already had this discussion.